### DISCUSSION TITLE
Anyone got a Solis S5 - EH1P5K-L working?

### DISCUSSION THREAD

**Discussion by ceemjay** [StephanJoubert/home_assistant_solarman#296] [Q&A]
I have followed the instructions and selected both solis yaml files but each entry on the dashboard is either Unknown or Unavailable.

New to HA so still learning how things hang together but I have found the logs which show this:

__Logger:_ custom_components.solarman.solarman
Source: custom_components/solarman/solarman.py:173
Integration: Solarman ([documentation](https://github.com/StephanJoubert/home_assistant_solarman/blob/main/README.md), [issues](https://github.com/StephanJoubert/home_assistant_solarman/issues))
First occurred: 18:37:30 (227 occurrences)
Last logged: 22:23:31

Querying registers [33029 - 33095] failed, aborting._

Any assistance would be most welcome - thanks

**Comment by TikTok90** [comment]
I have the Solis S5-GR1P3K inverter and the solis_1p8k-5g.yaml file worked for me
